code-studio/src/js/plc/task_creation.js

```diff
@@ -14,15 +14,17 @@ $(window).load(function () {
   });
 });
 
+// TODO: Don't use one monolothic js file
```

Comment:
Temporary measure - I need to break up plc stuff into multiple js files.

We were trying to create the IconLibrary on all pages, not just the ones that needed it. This is a workaround until the breakup happens.